Google.PowerShell.IntegrationTests/Storage/Storage.GcsObject.Tests.ps1

```diff
@@ -642,6 +642,12 @@ Describe "Copy-GcsObject" {
             Read-GcsObject $bucket "test-dest2" | Should Be "test2-$r"
         }
 
+        It "Should overwrite existing object" {
+            $sourceObj = Get-GcsObject $bucket "test-source"
+            $sourceObj | Copy-GcsObject $bucket "test-dest2" -Force
```

Comment:
Sorry to be a pain here, but could you confirm an error if you don't specify `-Force`? Otherwise it could be that we always overwrite, regardless of if you set the flag.

````
{ $sourceObj | Copy-GcsObject $bucket "should-fail" } |
    Should Throw 409  # Conflit
````